### PR TITLE
config: enable label plugins for pd

### PR DESCRIPTION
**Pull request by hi-rustin**
/hold 
Waiting the pd team review.

### CODE REVIEW

**Comment by hi-rustin** [comment]
@nolouch @rleungx PTAL~

File: prow/config/labels.yaml [hunk 1]
```diff
@@ -342,3 +358,531 @@ repos:
...
+      color: adf77b
+      description: The feature is under reviewing.
+      target: prs
+      prowPlugin: ti-community-label
+      isExternalPlugin: true
+      addedBy: anyone
+    - name: priority/P0
+      color: fc2929
+      description: The issue has P0 priority.
+      target: issues
```
Comment (rleungx) [inline prow/config/labels.yaml:717]: both for priority/xxx

File: prow/config/labels.yaml [hunk 2]
```diff
@@ -342,3 +358,531 @@ repos:
...
+      color: bfd4f2
+      description: relative cloud.
+      target: both
+      prowPlugin: ti-community-label
+      isExternalPlugin: true
+      addedBy: anyone
+    - name: epic/co-locality
+      color: f2c02b
+      description: relative data location.
+      target: both
```
Comment (rleungx) [inline prow/config/labels.yaml:682]: only issue is ok for epic/xxx

File: prow/config/labels.yaml [hunk 3]
```diff
@@ -342,3 +358,531 @@ repos:
...
+      color: ededed
+      description: "The issue's severity is critical".
+      target: both
+      prowPlugin: ti-community-label
+      isExternalPlugin: true
+      addedBy: anyone
+    - name: severity/major
+      color: ededed
+      description: "The issue's severity is major".
+      target: both
```
Comment (rleungx) [inline prow/config/labels.yaml:752]: only issue is ok for severity

Comment (hi-rustin) [reply prow/config/labels.yaml]: Fixed.

File: prow/config/labels.yaml (hunk 2 above)
Comment (hi-rustin) [reply prow/config/labels.yaml]: Fixed.

File: prow/config/labels.yaml (hunk 1 above)
Comment (hi-rustin) [reply prow/config/labels.yaml]: Fixed.

**Comment by hi-rustin** [comment]
@rleungx ping~

**Review by rleungx — APPROVED** [review]

**Comment by hi-rustin** [comment]
@rleungx Do you need the [require-matching-label](https://book.prow.tidb.io/plugins/require-matching-label.html) plugin?

It seems your issues require type label.

**Comment by rleungx** [comment]
> @rleungx Do you need the [require-matching-label](https://book.prow.tidb.io/plugins/require-matching-label.html) plugin?
> 
> It seems your issues require type label.

It will be better.

**Comment by hi-rustin** [comment]
> It will be better.

Added. Please take a look.

**Review by rleungx — APPROVED** [review]
LGTM

**Comment by hi-rustin** [comment]
/assign @Mini256 

PTAL~ Thanks!

File: prow/config/labels.yaml [hunk 4]
```diff
@@ -342,3 +358,538 @@ repos:
...
+      color: fbca04
+      description: Do not merge this PR.
+      target: prs
+      prowPlugin: ti-community-label
+      isExternalPlugin: true
+      addedBy: anyone
+    - name: status/TODO
+      color: ead79f
+      description: The issue will be done in the future.
+      target: prs
```
Comment (Mini256) [inline prow/config/labels.yaml:787]: The target seems to be issues

File: prow/config/labels.yaml [hunk 5]
```diff
@@ -342,3 +358,538 @@ repos:
...
+      prowPlugin: ti-community-label
+      isExternalPlugin: true
+      addedBy: anyone
+    - name: status/discussion-wanted
+      color: ead79f
+      description: The issue needs to be discussed.
+      target: prs
```
Comment (Mini256) [inline prow/config/labels.yaml:794]: ditto

File: prow/config/labels.yaml [hunk 6]
```diff
@@ -342,3 +358,538 @@ repos:
...
+      prowPlugin: ti-community-label
+      isExternalPlugin: true
+      addedBy: anyone
+    - name: status/invalid
+      color: e6e6e6
+      description: The issue is invalid.
+      target: prs
```
Comment (Mini256) [inline prow/config/labels.yaml:801]: ditto

File: prow/config/labels.yaml [hunk 7]
```diff
@@ -342,3 +358,538 @@ repos:
...
+      prowPlugin: ti-community-label
+      isExternalPlugin: true
+      addedBy: anyone
+    - name: status/needs-more-info
+      color: ededed
+      description: The issue needs more information to investigate.
+      target: prs
```
Comment (Mini256) [inline prow/config/labels.yaml:808]: ditto

File: prow/config/labels.yaml (hunk 4 above)
Comment (hi-rustin) [reply prow/config/labels.yaml]: Good catch!

File: prow/config/labels.yaml (hunk 6 above)
Comment (hi-rustin) [reply prow/config/labels.yaml]: Fixed.

**Pull request merged**